docs/openapi/tools-api.yaml

```diff
@@ -0,0 +1,118 @@
+import:
+  post:
+    tags:
+      - import
+    summary: Create a new import job
+    description: |
+      Note: Not implemented yet.
+      
+      This endpoint can be used to start a new import job with a set of URLs, import API key,
+      and an optional import.js mapping file. The import job will be processed asynchronously and 
+      the status of the job can be queried at the /tools/import/{jobId} endpoint. If there is no 
+      import queue available at the time of the request, a 503 Service Unavailable response will 
+      be returned.
+    operationId: startImportJob
+    security:
+      - api_key: [ ]
+    parameters:
+      - in: header
+        name: x-import-api-key
+        schema:
+          type: string
+        description: |
+          Client-specific import API key. This same key will be required to fetch the job status, 
+          and final .zip result.
+    requestBody:
+      required: true
+      content:
+        multipart/form-data:
```

Comment:
@solaris007, I had originally picked this content type such that we could easily accept a file (import.js) as part of the request. Now that I am writing some more tests, I see that `helix-shared-body-data` does not appear to support this content type: https://github.com/adobe/helix-shared/blob/main/packages/helix-shared-body-data/src/body-data-wrapper.js#L18-L19

How do you feel about using `application/json` instead, and including import.js as a Base64 encoded string?

Reply:
I've proposed this as a change for your consideration. The API spec seems to prefer this content type (it didn't render anything for the `multipart/form-data` request previously): 

<img width="1285" alt="Screenshot 2024-05-29 at 4 12 46 PM" src="https://github.com/adobe/spacecat-api-service/assets/1048207/b8d1fd1f-0cd1-4842-947b-12226be7fedb">

Reply:
@blefebvre i prefer the JSON approach as well. it is in line with helix standards.